src/cattrs/preconf/msgpack.py

```diff
@@ -30,6 +30,8 @@ def configure_converter(converter: BaseConverter):
     converter.register_structure_hook(
         datetime, lambda v, _: datetime.fromtimestamp(v, timezone.utc)
     )
+    converter.register_unstructure_hook(date, lambda v: v.isoformat())
```

Comment:
I don't know much about msgpack, but it seems like the msgpack converter is doing things a bit differently with `datetime` - it is using `timestamp` rather than `isoformat`. Is there a reason the msgpack converter does this differently? How should we be handling `date`s in the msgpack converter?

Reply:
We can, but let's make it use timestamps too, for consistency.

The idea is for the JSON converters to use the ISO format since JSON is meant to be somewhat human readable, and JavaScript can parse them out of the box. For binary formats like msgpack I think timestamps are a better fit since they're much more efficient.

Reply:
Interesting, thanks. Updated now. Since `date` has no `.timestamp` method, I've used the midnight-aligned datetime.